tests/class-searchpress-unit-test-case.php

```diff
@@ -41,14 +52,76 @@ function search_and_get_field( $args, $field = 'post_name' ) {
 		return sp_results_pluck( $posts, $field );
 	}
 
-	protected function refresh_index() {
+	protected static function refresh_index() {
 		SP_API()->post( '_refresh' );
 	}
 
-	protected function index( $posts ) {
+	protected static function index( $posts ) {
 		$posts = is_array( $posts ) ? $posts : [ $posts ];
 		SP_API()->index_posts( $posts );
-		$this->refresh_index();
+		self::refresh_index();
+	}
+
+	protected static function create_sample_content() {
+		$cat_a = self::factory()->term->create( array( 'taxonomy' => 'category', 'name' => 'cat-a' ) );
```

Comment:
Posts to index is an array of created posts - would it be useful to use the same data structure here, and have categories be an array of created categories?

Reply:
nah, posts are only collected so they can be indexed. putting the categories in an array would only make some of these too-long-lines even longer